apps/ledger-live-mobile/e2e/specs/speculos/addAccount/AddAccount.ts

```diff
@@ -0,0 +1,35 @@
+import { Application } from "../../../page";
+import { Currency } from "@ledgerhq/live-common/e2e/enum/Currency";
+
+export async function runAddAccountTest(currency: Currency, tmsLink: string) {
+  const app = new Application();
+
+  describe(`Add accounts - ${currency.name}`, () => {
+    beforeAll(async () => {
+      await app.init({
+        userdata: "onboardingcompleted",
```

Comment:
Maybe in the futur we can rename this userdata in `skip-onboarding` or the one in LLD in `onboardingcompleted`to have something homogeneous :)